po/extra/nl.po

```diff
@@ -0,0 +1,76 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-05-20 22:00+0900\n"
+"PO-Revision-Date: 2021-05-20 14:31+0200\n"
+"Last-Translator: Heimen Stoffels <vistausss@fastmail.com>\n"
+"Language-Team: \n"
+"Language: nl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.2\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: data/louper.appdata.xml.in:8 data/louper.desktop.in:3
+msgid "Louper"
+msgstr "Vergrootglas"
+
+#: data/louper.appdata.xml.in:9 data/louper.desktop.in:5
+msgid "Magnify the selected text"
+msgstr "Vergroot geselecteerde tekst"
+
+#: data/louper.appdata.xml.in:11
+msgid "A text magnification app that helps you zoom the selected text."
+msgstr "Een vergrootglastoepassing om geselecteerde tekst mee te vergroten."
+
+#: data/louper.appdata.xml.in:14
+msgid "Workflow:"
+msgstr "Werkmethode:"
+
+#: data/louper.appdata.xml.in:16
+msgid "Select some text"
+msgstr "Selecteer tekst"
+
+#: data/louper.appdata.xml.in:17
+msgid ""
+"Launch the app (it is recommended to assign a shortcut key in Keyboard "
+"Settings). The app shows the selected text in huge size"
+msgstr ""
```

Comment:
FYI this is where I'd like you to add a translation :)

Reply:
Done! 🙂